belfast.md

```diff
@@ -0,0 +1,4 @@
+### Belfast, Northern Ireland
+
+ - Filming location for Game of Thrones
+ - Go to eat at Burger Shack
```

Comment:
Is that the same as the Love Shack or is that different?
Do you like that song though @scottmcelveen ?
What do you give it out of :100: